rllib/agents/dqn/tests/test_apex_dqn.py

```diff
@@ -62,6 +63,64 @@ def test_apex_dqn_compilation_and_per_worker_epsilon_values(self):
 
             trainer.stop()
 
+    def test_apex_lr_schedule(self):
```

Comment:
Nice test! Just concerned that 30 iters take some time. Should we either a) only run 5 iters per _step_n_times call or b) make the test larger (currently size=medium in BUILD). Otherwise, the test will flake out a lot, I think.